CHANGELOG.md

```diff
@@ -1,3 +1,130 @@
+- Update `monokai-pro-spectrum` ([#3814](https://github.com/helix-editor/helix/pull/3814))
+- Update `Nord` ([#3792](https://github.com/helix-editor/helix/pull/3792)
+- Update `fleetish` ([#3844](https://github.com/helix-editor/helix/pull/3844))
+- Update `flatwhite` ([#3843](https://github.com/helix-editor/helix/pull/3843))
+- Add `darcula` ([#3739](https://github.com/helix-editor/helix/pull/3739))
+- Automatically lint themes and update existing themes ([#3234](https://github.com/helix-editor/helix/pull/3234), [#3587](https://github.com/helix-editor/helix/pull/3587)))
+- Add bufferline colors to 15 themes ([#3881](https://github.com/helix-editor/helix/pull/3881))
+- Add `gruvbox_dark_hard` ([#3948](https://github.com/helix-editor/helix/pull/3948))
+- Add `onedarker` ([#3980](https://github.com/helix-editor/helix/pull/3980))
+- Add ability to derive ([#3067](https://github.com/helix-editor/helix/pull/3067))
+- Add `dark_high_contrast` ([#3312](https://github.com/helix-editor/helix/pull/3312))
+- Update `bogster` ([#4121](https://github.com/helix-editor/helix/pull/4121))
+- Update `sonokai`([#4089](https://github.com/helix-editor/helix/pull/4089))
+
+LSP:
+
+- Switch to `openscad-lsp` for OpenSCAD ([#3750](https://github.com/helix-editor/helix/pull/3750))
+- Markdown ([#3499](https://github.com/helix-editor/helix/pull/3499))
+- Jsonnet ([#3714](https://github.com/helix-editor/helix/pull/3714))
+- Bass ([#3771](https://github.com/helix-editor/helix/pull/3771))
+
+New languages:
+
+- SML ([#3692](https://github.com/helix-editor/helix/pull/3692))
+- Jsonnet ([#3714](https://github.com/helix-editor/helix/pull/3714))
+- Godot-resource ([#3759](https://github.com/helix-editor/helix/pull/3759))
+- Astro ([#3829](https://github.com/helix-editor/helix/pull/3829))
+- Bass ([#3771](https://github.com/helix-editor/helix/pull/3771))
+- Webassembly text format ([#4040](https://github.com/helix-editor/helix/pull/4040))
+
+
+Updated languages and queries:
+
+- Zig ([#3621](https://github.com/helix-editor/helix/pull/3621))
+- Solidity
+- Fish ([#3704](https://github.com/helix-editor/helix/pull/3704))
+- Elixir ([#3645](https://github.com/helix-editor/helix/pull/3645))
+- Update Git-Diff and rename to Diff ([#3708](https://github.com/helix-editor/helix/pull/3708))
+- Add language injection for Nix ([#3594](https://github.com/helix-editor/helix/pull/3594))
+- Fix highlight rules for Nix ([#3697](https://github.com/helix-editor/helix/pull/3697))
+- Add highlights for attributes for Rust ([#3729](https://github.com/helix-editor/helix/pull/3729))
+- Show rulers while editing Git-Commit messages ([#3738](https://github.com/helix-editor/helix/pull/3738))
+- Add `.markdown` as recognized extension for Markdown ([#3749](https://github.com/helix-editor/helix/pull/3749))
+- Render HTML `<code>` tags as code in Markdown ([#3425](https://github.com/helix-editor/helix/pull/3425))
```

Comment:
Quick note, this PR should  not be in the languages category it should be in Usability improvement and fixes. The markdown it's referring to is the markdown in LSP help popups.
(By having the PR authors write the CHANGELOG like you said this kind of stuff could be avoided :P)